code/system/application/sensation/include/geolocation/geolocation.hpp

```diff
@@ -52,9 +52,37 @@ class Geolocation
   void setUp();
   void tearDown();
 
+  /**
+    *  Accuracy of the geographical position m
+    *
+    */
+  double calculatePositionConfidence();
+
+
+  /** The heading confidence in rad
+    */
+  double calculateHeadingConfidence();
+
+  /**   The heading rate in rad/s.
+    *
+    */
+  double calculateHeadingRateConfidence();
+
+  /**   Speed confidence in m/s.
+    *
+    */
+   double calculateSpeedConfidence();
+
 
    Kalman::ExtendedKalmanFilter<opendlv::sensation::geolocation::State<double>> 
        m_ekf;
+
+
+   const double m_gpsToCoGDisplacement_x = -1.5; ///--> Displacement between the real position of the GPS and the CoG of the vehicle in [m]
```

Comment:
The name is not OK, should be m_gpsToCogDisplacementX  (two changes). There should not be constants here, read from config instead.